spec/factories/trainees.rb

```diff
@@ -1,17 +1,51 @@
 FactoryBot.define do
   factory :trainee do
+    sequence(:trainee_id) { |n| n.to_s }
+    first_names { Faker::Name.first_name }
+    last_name { Faker::Name.last_name }
+    gender { %w[Female Male].sample }
```

Comment:
```suggestion
    gender { %w[Female Male Other].sample }
```

![image](https://user-images.githubusercontent.com/470137/91959379-6fd16c00-ed00-11ea-969b-5a58b80c6a23.png)

Reply:
good spot, done